app/web/cypress/e2e/modelling-functionality/create-component.cy.ts

```diff
@@ -13,6 +13,7 @@ Cypress._.times(SI_CYPRESS_MULTIPLIER, () => {
     beforeEach(function () {
       //cy.setupVariables();
       cy.loginToAuth0(AUTH0_USERNAME, AUTH0_PASSWORD);
+      cy.log("fuck yeah i logged in!");
```

Comment:
Probably not required!

Reply:
ah whoops i definitely meant to remove that 😅